### PR TITLE
[GAWB-2070] Javascript error and not able to navigate between data ta…

### PR DESCRIPTION
**Pull request by rtitle**
…bs after importing attribute file

https://broadinstitute.atlassian.net/browse/GAWB-2070

**Problem summary:**
Importing workspace column defaults with the wrong JSON schema was causing Javascript errors in the Data tab. I added some basic JSON schema validation to the `workspace-column-defaults` attribute. Discussed with PO whether to reject bad data immediately at import time or silently accept it and choose reasonable defaults -- they preferred the latter.

- [x] **Submitter**: Include the JIRA issue number in the PR description
- [x] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [x] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * To Demo flag is set
  * Release Summary is filled out, if applicable
  * Add notes on how to QA
- [x] **Submitter**: Update RC_XXX release ticket with any config or environment changes necessary
- [x] **Submitter**: Update FISMA documentation if changes to:
  * Authentication
  * Authorization
  * Encryption
  * Audit trails
- [x] **Submitter**: If you're adding new libraries, sign us up to security updates for them
- [x] Tell the tech lead (TL) that the PR exists if they wants to look at it
- [x] Anoint a lead reviewer (LR). **Assign PR to LR**
* Review cycle:
  * LR reviews
  * Rest of team may comment on PR at will
  * **LR assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to LR** for further feedback
- [x] **TL** sign off
- [x] **LR** sign off
- [x] **Product Owner** sign off
- [x] **Assign to submitter** to finalize
- [x] **Submitter**: Verify all tests go green, including CI tests
- [x] **Submitter**: Squash commits and merge to develop
- [x] **Submitter**: Delete branch after merge
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

**Comment by rtitle** [comment]
@zarsky-broad sorry to keep bothering you with these PRs

**Comment by zarsky-broad** [comment]
¯\\_(ツ)_/¯

File: src/cljs/main/broadfcui/page/workspace/data/tab.cljs [hunk 1]
```diff
@@ -104,11 +104,7 @@
          {:ref "entity-table"
           :workspace-id workspace-id
           :column-defaults
-          (try
-            (utils/parse-json-string (get-in workspace [:workspace :workspace-attributes
-                                                        :workspace-column-defaults]))
-            (catch js/Object e
-              (utils/jslog e) nil))
+          (data-utils/get-column-defaults workspace)
```
Comment (zarsky-broad) [inline src/cljs/main/broadfcui/page/workspace/data/tab.cljs]: Why not do the `get-in` here, rather than passing the entire workspace?

**Review by zarsky-broad — APPROVED** [review]
One comment, looks good.

File: src/cljs/main/broadfcui/page/workspace/data/tab.cljs (hunk 1 above)
Comment (rtitle) [reply src/cljs/main/broadfcui/page/workspace/data/tab.cljs]: No reason, that sounds good :)

**Pull request merged**